qa/tasks/cephfs/test_admin.py

```diff
@@ -17,7 +18,7 @@ class TestAdminCommands(CephFSTestCase):
     """
 
     CLIENTS_REQUIRED = 1
-    MDSS_REQUIRED = 1
+    MDSS_REQUIRED = 2
```

Comment:
I think it would be better to group the tests for `fs rename` under a separate class so that the entire group can be executed in a single go.

Reply:
Done